lib/linter-registry.js

```diff
@@ -66,10 +66,10 @@ export default class LinterRegistry {
     const filePath = editor.getPath()
 
     if (
-      (onChange && !this.lintOnChange) ||                                          // Lint-on-change mismatch
-      !filePath ||                                                                 // Not saved anywhere yet
-      isPathIgnored(editor.getPath(), this.ignoreGlob, this.ignoreVCS) ||          // Ignored by VCS or Glob
-      (!this.lintPreviewTabs && editor.hasTerminatedPendingState === false)        // Ignore Preview tabs
+      (onChange && !this.lintOnChange) ||                                                       // Lint-on-change mismatch
+      !filePath ||                                                                              // Not saved anywhere yet
+      isPathIgnored(editor.getPath(), this.ignoreGlob, this.ignoreVCS) ||                       // Ignored by VCS or Glob
+      (!this.lintPreviewTabs && atom.workspace.getActivePane().getPendingItem() === editor)     // Ignore Preview tabs
```

Comment:
Might be best to follow the logic `tab-view` [uses here](https://github.com/atom/tabs/blob/4c94444616f0ea6f851f62fd825c9c114669e988/lib/tab-view.coffee#L231).